### ISSUE TITLE
PP UX/training confusion re: loss = -1.  (need to better document or add auto logging of last rank loss?) 

### ISSUE THREAD

**Issue by lessw2020** [pytorch/torchtitan#550]
For PP, only the last rank reports the loss. 
However, in titan, by default we only log the first rank, rank 0.
This results in UX and user confusion in that doing a simple run with PP will report an ongoing loss of -1....

The fix is to ensure that the last rank is also being logged, where last rank is N-1 with N = num gpus.

i.e. in the run_llama_train.sh:
~~~
LOG_RANK=${LOG_RANK:-0}
~~~
will report loss = -1 for 8 gpu run.
The manual fix is:
~~~
LOG_RANK=${LOG_RANK:-0,7}
~~~

However, how to do this automatically is unclear.
This issue is mostly to track this and discuss potential fixes to ensure reduced user confusion when running PP, ala avoiding user experience like this issue:
https://github.com/pytorch/torchtitan/issues/547
